packages/core/src/scp-cf/destination.ts

```diff
@@ -93,31 +105,23 @@ function parseAuthTokens(destination: MapType<any>): MapType<any> {
   return assoc('authTokens', authTokens, destination);
 }
 
-function setTrustAll(destination: MapType<any>): MapType<any> {
+function setTrustAll(destination: Destination): Destination {
   return assoc('isTrustingAllCertificates', parseTrustAll(destination.isTrustingAllCertificates), destination);
 }
 
-function parseTrustAll(isTrustingAllCertificates: string | boolean): boolean {
-  if (!isTrustingAllCertificates) {
-    return false;
-  } else if (typeof isTrustingAllCertificates === 'boolean') {
-    return isTrustingAllCertificates;
-  } else {
+function parseTrustAll(isTrustingAllCertificates?: string | boolean): boolean {
+  if (typeof isTrustingAllCertificates === 'string') {
     return isTrustingAllCertificates.toLowerCase() === 'true';
   }
+
+  return !!isTrustingAllCertificates;
```

Comment:
what is `!!` two times -1 is +1?

Reply:
`!!true => true`
`!!false => false`
`!!undefined => false` (this is the case for anything falsy)

Reply:
so the ? in the signature allows for undefined?  This seems like dark magic, but the experience JS wizzards do not see it that way I guess....

Reply:
what about using `isTrustingAllCertificates ?? false` this would be a bit more explicit that in the nullish case we return false. However, if the `!!` is a well established concept in the JS scene then keep it.

Reply:
`!!` is well established and a little easier in this case I think. But I keep forgetting about `??`, thanks for the reminder 😊